### ISSUE TITLE
Average loss calculation

### ISSUE THREAD

**Issue by sriprabhar** [facebookresearch/fastMRI#19]
Thanks for posting the fastMRI code. Would like to know if there is any specific reason for choosing the following weighting scheme for computing the average loss.

avg_loss = 0.99 * avg_loss + 0.01 * loss.item() if iter > 0 else loss.item()

Thanks in advance for your reply.

**Comment by anuroopsriram** [comment]
It's simply computing an exponential moving average: https://en.wikipedia.org/wiki/Exponential_smoothing
